Xamarin.Forms.Core/Markup/DefaultBindableProperties.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Xamarin.Forms.Markup
+{
+	public static class DefaultBindableProperties
```

Comment:
if I only make a few strong statements in this review, this will be one of it. This seems a nice idea, but hides too much. we don't have, and I don't want "Default" properties. The only default we have right now is defined by the ContentProperty...

Reply:
Developers do want default properties.

E.g. compare:
```C#
new Image { }
           .Bind (nameof(vm.Icon)),

new Label { LineBreakMode = LineBreakMode.WordWrap }
           .Bind (nameof(vm.RegistrationPrompt)),
```
to:
```C#
new Image { }
           .Bind (Image.SourceProperty, nameof(vm.Icon)),

new Label { LineBreakMode = LineBreakMode.WordWrap }
           .Bind (Label.TextProperty, nameof(vm.RegistrationPrompt)),
```
 
A default property is only defined for those types where there is a **very** obvious choice.

If devs don't like using (some) defaults, they can specify the property explicitly as above. Robbing devs of the option to use defaults seems like forcing them to type (and more importantly read) a lot of noise repetition.